components/Container/containers.scss

```diff
@@ -13,7 +13,7 @@
   position: relative;
   width: 100%;
   padding: var(--layout-content-padding, var(--brand-space2));
-  background-color: var(--layout-content-background, white);
+  background-color: var(--layout-content-background);
```

Comment:
Removed the default fallback to white background, since I found myself overriding it for every page I was working on